heltour/tournament/admin.py

```diff
@@ -661,7 +661,9 @@ def player_info_view(self, request, object_id, player_name):
             'league': season.league,
             'player': season_player.player,
             'reg': reg,
-            'has_played_20_games': has_played_20_games
+            'has_played_20_games': has_played_20_games,
+            'date_created': sp.registration.friends if sp.registration else None,
+	    'friends': (sp.registration.date_created if sp.registration else sp.date_created).isoformat()
```

Comment:
Looks better, but I think there is a tab mixed in with the spaces here.  Can you verify that you're using spaces only for indentation? 4 spaces per level of indentation is what is proposed by PEP-8 (the python formatting guidelines) and is what we use in this project.